web/src/router.ts

```diff
@@ -104,9 +104,47 @@ const routes: RouteRecordRaw[] = [
           {
             path: 'settings',
             name: 'repo-settings',
-            component: (): Component => import('~/views/repo/RepoSettings.vue'),
+            component: (): Component => import('~/views/repo/settings/RepoSettings.vue'),
```

Comment:
Component can not be used with redirect simultaneously? Otherwise we could let `repo-settings` redirect to `repo-settings-general`.

Reply:
`name: 'repo-settings',` is basically not needed anymore, right? If it is, you could rename 'repo-settings-general' to it.

Reply:
The wrapper does not need a name anymore as it has a child with a default route:

```
path: '',
   name: 'repo-settings-general',
```.

You can rename that default child to `repo-settings` or change all router links to point to the new route ` repo-settings-general`